core/src/main/kotlin/builder/kord/ProxyKordBuilder.kt

```diff
@@ -0,0 +1,84 @@
+package dev.kord.core.builder.kord
+
+import dev.kord.cache.api.DataCache
+import dev.kord.common.annotation.KordExperimental
+import dev.kord.common.entity.Snowflake
+import dev.kord.core.ClientResources
+import dev.kord.core.Kord
+import dev.kord.core.exception.KordInitializationException
+import dev.kord.core.gateway.DefaultMasterGateway
+import dev.kord.core.supplier.EntitySupplyStrategy
+import dev.kord.gateway.Gateway
+import dev.kord.gateway.builder.Shards
+import dev.kord.rest.ratelimit.ExclusionRequestRateLimiter
+import dev.kord.rest.request.KtorRequestHandler
+import dev.kord.rest.request.RequestHandler
+import dev.kord.rest.service.RestClient
+import io.ktor.client.HttpClient
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.MutableSharedFlow
+
+/**
+ * The rest only Kord builder. You probably want to invoke the [DSL builder][Kord.restOnly] instead.
+ */
+@KordExperimental
+public class KordProxyBuilder {
```

Comment:
This is almost the same as `KordRestOnlyBuilder`, I think it would be better to have a common base class instead of copy pasting.

I think we should also have a similar thing like the `Kord.restOnly()` function for this builder.